inspirehep/modules/disambiguation/ext.py

```diff
@@ -52,6 +52,10 @@ def init_config(self, app):
             disambiguation_base_path, 'ethnicity.csv')
         app.config['DISAMBIGUATION_ETHNICITY_MODEL_PATH'] = os.path.join(
             disambiguation_base_path, 'ethnicity.pkl')
+        app.config['DISAMBIGUATION_SAMPLED_PAIRS_PATH'] = os.path.join(
+            disambiguation_base_path, 'pairs.json')
+        app.config['DISAMBIGUATION_DISTANCE_MODEL_PATH'] = os.path.join(
+            disambiguation_base_path, 'linkage.dat')
```

Comment:
Let's be consistent with the naming (and deviate from the paper) and call it `distance.pkl`.